### PR TITLE
V14: Documenting your controllers

### PR DESCRIPTION
**Pull request by Zeegaan**
## Description

Added documentation about how to document your controllers in swagger

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [x] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)
V14

## Deadline (if relevant)

Release of v14 🚀

### CODE REVIEW

**Comment by eshanrnh** [comment]
Thanks, @Zeegaan for the PR. We will takw a look and merge it in for the v14 release 💪

**Comment by alina-tincas** [comment]
Hi @Zeegaan before we review this PR, can you please add this new article to summary as well? Also, it seems you are adding the "documenting your controllers" in this PR as well, even though there is another PR (https://github.com/umbraco/UmbracoDocs/pull/6106) for this? 🙈

**Comment by Zeegaan** [comment]
@alina-tincas Yep, we need to move this around, so will do when i move it 😁 
And yea, I expected https://github.com/umbraco/UmbracoDocs/pull/6106 to be merged already, so I have the changes on this branch too, woops 🙈

**Comment by alina-tincas** [comment]
All righty, then in this case I think @jonat123 can review this PR and close the other one to not cause merge conflicts between PRs. @jonat123 what do you think? 😊

File: 14/umbraco-cms/fundamentals/backoffice/document-your-own-api-in-swagger.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+If we implement the api from the [Creating your own API](./create-your-own-api.md) article, it is originally under the `Management API` section. If we want to hove that, we can replace the `ManagementApiBaseController` attribute with a `Controller` attribute.
+```csharp
+[ApiController]
+[VersionedApiBackOfficeRoute("my/item")]
+[ApiExplorerSettings(GroupName = "My item API")]
+public class MyItemApiController : Controller
+```
+When you open swagger, it will show up under the `Default API` section.
+![Now it's under default API](images/DefaultListOfAPi.png)
+This is great if you only have 1 API, but if you have multiple, you might want to organise these as well as their own separate silos.
```
Comment (jonat123) [inline 14/umbraco-cms/fundamentals/backoffice/document-your-own-api-in-swagger.md:13]:
```suggestion
It's great if you only have 1 API, however, if you have multiple, you might want to organize these as well as their own separate silos.
```

File: 14/umbraco-cms/fundamentals/backoffice/document-your-own-api-in-swagger.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+﻿# Documenting your own API in swagger
+When you open swagger, you are met by a list of 3 APIs:
+![Default list of apis](images/DefaultListOfAPi.png)
+If we implement the api from the [Creating your own API](./create-your-own-api.md) article, it is originally under the `Management API` section. If we want to hove that, we can replace the `ManagementApiBaseController` attribute with a `Controller` attribute.
```
Comment (jonat123) [inline 14/umbraco-cms/fundamentals/backoffice/document-your-own-api-in-swagger.md:4]:
```suggestion
If we implement the API from the [Creating your own API](./create-your-own-api.md) article, it is originally under the `Management API` section. If we want to move that, we can replace the `ManagementApiBaseController` attribute with a `Controller` attribute.
```

**Comment by Zeegaan** [comment]
@jonat123 Should be ready now, it is now moved under the tutorial section instead 😁

**Pull request merged**